editor/src/clj/editor/properties.clj

```diff
@@ -599,9 +599,15 @@
 (defn round-vec-coarse [v]
   (mapv round-scalar-coarse v))
 
-(defn ->choicebox [vals]
-  {:type :choicebox
-   :options (mapv (juxt identity identity) (sort eutil/natural-order vals))})
+(defn ->choicebox
+  ([vals]
+   (->choicebox vals true))
+  ([vals apply-natural-sorting?]
+   (let [sorted-vals (if apply-natural-sorting?
+                       (sort eutil/natural-order vals)
+                       vals)]
+     {:type :choicebox
+      :options (mapv (juxt identity identity) sorted-vals)})))
```

Comment:
One more place where we have sorting (like the third one in this flow). So, I make it optional too.

Reply:
> Hmm, I really liked how you made it possible to supply a `sort-fn` as an argument. Maybe this could take a `sort-fn` argument as well? And if you supply `nil`, we don't sort the list?

Yes, that's how I implemented it in the first attempt. But! We are adding empty string`""` which isn't in the initial `layer->index` which makes my custom sorting function too complicated (not as beautiful as `(partial sort-by layer->index)`)